src/lightkurve/io/eleanorlite.py

```diff
@@ -37,9 +37,9 @@ def read_eleanorlite_lightcurve(filename,
         be used to mask out bad cadences. If a string is passed, it has the
         following meaning:
             * "none": no cadences will be ignored (`quality_bitmask=0`).
-            * "default": cadences with flags indicating AttitudeTweak, SafeMode, CoarsePoint, EarthPoint, Desat, or ManualExclude will be ignored
-            * "hard": cadences with default flags, ApertureCosmic, CollateralCosmic, Straylight, or Straylight2 will be ignored
-            * "hardest": TBD: cadences with all the above flags will be ignored, in addition to cadences with GSFC-ELEANOR-LITE bit flags of 17 or 18. This is done by setting both of these flags to be equal to ManualExclude = 128
+            * "default": cadences with flags indicating AttitudeTweak, SafeMode, CoarsePoint, EarthPoint, Desat, or ManualExclude will be ignored.
+            * "hard": cadences with default flags, ApertureCosmic, CollateralCosmic, Straylight, or Straylight2 will be ignored.
+            * "hardest": cadences with all the above flags will be ignored, in addition to cadences with GSFC-ELEANOR-LITE bit flags of 16 or 17.
```

Comment:
Describing them as 16/17 (or 17/18) only is probably not very clear. The conversation so far proves that.

I suggest we describe them as bit 17 (decimal value 65536) and bit 18 (decimal value 131072) . 

I say bit 17/18 rather than 16/17 so that it is consistent with the convention used in [TESS documentation](https://outerspace.stsci.edu/display/TESS/2.0+-+Data+Product+Overview).

---
Overall, we probably want to state something to the effect that Eleanor uses the TESS SPOC quality flags. Additionally, it adds bit 17 and bit 18. 

For most users, who are familiar with standard TESS SPOC products, it'd be easier to understand.

Reply:
The docstring looks good to me. From my end, the only outstanding issue is the testing of quality bitmasks. ( I've commented on `test_eleanorlite.py`, in case they get lost among the messages. )